ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -539,6 +539,36 @@ def EuclideanNorm(*, input, axis, keep_dims=False, name="EuclideanNorm"):
 ConcatV2 = to_ivy_arrays_and_back(map_raw_ops_alias(tf_frontend.concat))
 
 
+@to_ivy_arrays_and_back
+def Conv2D(
+    *,
+    input,
+    filter,
+    filter_shape,
```

Comment:
TensorFlow doesn't have an argument filter_shape - https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv2D

Reply:
okey for that now I have added it manually by   ivy.variable(ivy.random_normal(shape=filter_shape, stddev=0.1))

Reply:
Here we can't make changes to the arguments because we are accepting a Tensorflow function. The args must match exactly what TensorFlow specifies for Conv2D